frontend/src/pull.ts

```diff
@@ -115,10 +115,14 @@ export class Pull extends PullData {
     return (
       this.hasMetDeployRequirements() &&
       !this.getDevBlock() &&
-      (!!this.getDeployBlock() || !this.isCiRequired() || this.hasMergeConflicts())
+      (!!this.getDeployBlock() || !this.isCiRequired() || this.hasMergeConflicts() || this.isDependent())
     );
   }
 
+  isDependent(): boolean {
+    return !(this.base.ref == "main" || this.base.ref == "master");
```

Comment:
This seems good. Have we validated that when the dependent pull is merged github fires a `pull_updated` webhook for this pull when it's `base.ref` is updated?

I know github will auto-change it to the main branch when a dependent pull is merged, but I don't know if they send a webhook, though I suppose we could just merge this and find out later.

Reply:
Nice to know! Yeah, I assumed their API wouldn't lie about the current state; I was less sure if they send a webhook that the pull has changed otherwise pulldasher wouldn't know till someone clicked the refresh button on the pull.

But I think we should merge this anyway and deal with it if run into the problem.